source/geod24/concurrency.d

```diff
@@ -941,19 +941,19 @@ class FiberScheduler
      * This creates a new Fiber for the supplied op and then starts the
      * dispatcher.
      */
-    void start(void delegate() op)
+    void start(void delegate() op, size_t sz = 0)
```

Comment:
You could just use the actual default value here.
My suggestion would be to have, somewhere:
```D
///
private enum DefaultFiberStackSize = 16 * 1024 * 1024;
```

And use this as default value here.